uber/templates/emails/mivs/accepted/2019_HandbookTrainingUpdate.txt

```diff
@@ -1,6 +1,6 @@
-Ahoy {{ game.studio.primary_contact.first_name }},
+Ahoy {{ guest.group.studio.primary_contact_first_names }},
```

Comment:
This was a copy-paste error -- this email uses the GuestGroup model, so we need to start from the GuestGroup model instead of the IndieGame model.